resources/ory/values.yaml

```diff
@@ -80,13 +80,14 @@ hydra:
     dangerousForceHttp: true
     dangerousAllowInsecureRedirectUrls: false
     existingSecret: "ory-hydra-credentials"
-  resources:
-    limits:
-      cpu: 100m
-      memory: 256Mi
-    requests:
-      cpu: 50m
-      memory: 128Mi
+    deployment:
```

Comment:
resources: are now under hydra.hydra., it should be moved one level upper to hydra.

Reply:
ooups, fixed